src/cloud/project/services-elastic.md

```diff
@@ -199,5 +199,38 @@ elasticsearch:
 If you use the ElasticSuite third-party plugin, you must [update the `{{site.data.var.ct}}` package]({{ site.baseurl }}/cloud/project/ece-tools-update.html) to version 2002.0.19 or later.
 When setting up ElasticSuite, add the configuration settings to the `ELASTICSUITE_CONFIGURATION` deploy variable. This configuration saves the settings across deployments.
 
+### Remove plugins for Elasticsearch
+Removing the plugin entries from `elasticsearch:` in `.magento/services.yaml` does not uninstall or disable them as you might expect. You must take the additional step of reindexing your Elasticsearch data. This behavior is intentional to prevent possible loss or corruption of data that depends on these plugins.
+
+**To remove Elasticsearch plugins:**
```

Comment:
@bdenham The review is over, but in the future this needs to be formatted using the procedure heading callout.